split.js

```diff
@@ -1,33 +1,44 @@
 var utils = require('./utils')
+var BN = require('bn.js')
+var ext = require('./bn-extensions')
 
 // split utxos between each output, ignores outputs with .value defined
 module.exports = function split (utxos, outputs, feeRate) {
-  if (!isFinite(utils.uintOrNaN(feeRate))) return {}
+  if (!isFinite(utils.bnOrNaN(feeRate))) return {}
 
   var bytesAccum = utils.transactionBytes(utxos, outputs)
-  var fee = feeRate * bytesAccum
+  var fee = ext.multiply(feeRate, bytesAccum)
   if (outputs.length === 0) return { fee: fee }
 
   var inAccum = utils.sumOrNaN(utxos)
   var outAccum = utils.sumForgiving(outputs)
-  var remaining = inAccum - outAccum - fee
+  var remaining = ext.subtract(ext.subtract(inAccum, outAccum), fee)
```

Comment:
`add` `sub` `mul` `div` `lt` `lg` `lte` etc?

Reply:
those are pretty standard names, not used just by `bn`. see this refactor: https://github.com/cosmos/cosmos-sdk/commit/a814e5ce66e4068306733b8b97411001feb8ce72